lightly/active_learning/utils/bounding_box.py

```diff
@@ -68,6 +82,16 @@ def from_x_y_w_h(cls, x: float, y: float, w: float, h: float):
         """
         return cls(x, y, x + w, y + h)
 
+    @classmethod
+    def from_yolo(cls, x_center: float, y_center: float, w: float, h: float):
```

Comment:
I'm not sure we should add these methods for all kinds of formats. There will be label format and prediction formats. We can use this as a temporary solution now because we want to quickly prototype this. But I would rethink how we can do this in a clean and scalable way.

I would for now rename `from_yolo` to `from_yolo_annotation` or `from_yolo_label` just to make clear that this is the label format not the output of the model or prediction.

I would maybe add an additional line to the description illustrating what happens here?
```python
# x_center, y_center, w, h --> x0, y0, x1, y1
```